app/components/candidate_interface/volunteering_review_component.html.erb

```diff
@@ -15,13 +15,12 @@
 <% if show_missing_banner? %>
   <%= render(CandidateInterface::IncompleteSectionComponent.new(section: :volunteering, section_path: candidate_interface_volunteering_experience_path, error: @missing_error)) %>
 <% elsif @application_form.application_volunteering_experiences.empty? %>
-  <section class="app-summary-card govuk-!-margin-bottom-6">
-    <%= render(SummaryCardHeaderComponent.new(title: t('application_form.volunteering.no_experience.summary_card_title'), heading_level: @heading_level)) %>
+  <div class="govuk-inset-text govuk-!-margin-top-0">
```

Comment:
```suggestion
  <div class="govuk-inset-text govuk-!-margin-top-0 govuk-!-width-two-thirds">
```

And then remove the grid row and column wrapper divs that surround this and the page heading.

Reply:
Should this be using the `<%= govuk_inset_text(classes: 'govuk-!-margin-top-0 govuk-!-width-two-thirds') do %>` pattern established here?